src/main/java/Helper.java

```diff
@@ -0,0 +1,122 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.ArrayList;
+
+public class Helper extends Duke{
+    public static void addToDo(String[] line, ArrayList<Task> taskList) throws DukeException {
+        String description = line[1];
+        // Validate description
+        if (description.isEmpty()) {
+            throw new DukeException("☹ OOPS!!! Description of todo unclear. Here's an example: 'todo borrow book'");
+        }
+        Task task = new ToDo(description);
+        taskList.add(task);
+        ReadWriteFile.writeToFile(taskList);
+
+        System.out.println("Got it. I've added this task:");
```

Comment:
You may wish to consider combining the 3 "System.out.println(..)" into one e.g. 
```
System.out.println("Got it. I've added this task:" 
                    + task + "Now you have "
                    + taskList.size() + " task(s) in the list.");
```

Reply:
Oh right, thanks for the code example as well. It sure helps! 👍